src/lib/services/classifiersService.ts

```diff
@@ -7,12 +8,32 @@ export interface ClassifiersData {
     brands: string[];
     colors: string[];
     priceIntervals: string[];
+    productVariants: ProductVariant[];
 }
 
 export const getAllClassifiers = async (): Promise<ClassifiersData> => {
-    const { request } = classifiersService.getAll<ClassifiersData>();
-    const response = await request;
-    return Array.isArray(response.data) ? response.data[0] : response.data;
+    const { request: classifiersRequest } =
+        classifiersService.getAll<ClassifiersData>();
```

Comment:
here I suggest to refactor the `getAll` method in the `httpService` to handle the response from the classifiers endpoint better. 
Also I do not understand why is it necesarry to make a call to products endpoint here, but maybe I am missing something.